### PR TITLE
Add display config to shells (#268)

### PR DESCRIPTION
**Pull request by jermainegug**
This pull request solves issue #268. Potentially could add options to display specific parts of the config file.

### CODE REVIEW

**Comment by wtgee** [comment]
@jamessynge you added this issue, anything else you had in mind?

File: bin/pocs_shell [hunk 1]
```diff
@@ -7,6 +7,7 @@ import zmq
 from cmd import Cmd
 from subprocess import TimeoutExpired
 from pprint import pprint
+from pprint import pprint
```
Comment (wtgee) [inline bin/pocs_shell]: Looks like this is already there?

File: bin/pocs_shell [hunk 2]
```diff
@@ -74,6 +74,13 @@ class PocsShell(Cmd):
 
         return self.pocs.is_safe()
 
+    def do_display_config(self, *arg):
+        """ Display the config file for POCS """
+        if self.pocs.config is None:
```
Comment (jamessynge) [inline bin/pocs_shell]: What if self.pocs is not set (i.e. setup_pocs has not been run)? Seems like it would throw an exception.

**Comment by jamessynge** [comment]
Thanks very much for this change, I look forward to using it.

File: bin/pocs_shell (hunk 2 above)
Comment (wtgee) [reply bin/pocs_shell]: @jermainegug Can you check on the above?

File: bin/peas_shell [hunk 3]
```diff
@@ -123,6 +124,13 @@ class PanSensorShell(cmd.Cmd):
             except Exception as e:
                 print_error("Can't change log level for {}".format(sensor))
 
+    def do_display_config(self, config):
+        """ Display the config file for PEAS """
+        if self.config=={}:
```
Comment (wtgee) [inline bin/peas_shell]: The way to check for this would be to just say `if self.config`:

File: bin/pocs_shell [hunk 4]
```diff
@@ -74,6 +74,17 @@ class PocsShell(Cmd):
 
         return self.pocs.is_safe()
 
+    def do_display_config(self, *arg):
+        """ Display the config file for POCS """
+        try:
+            if self.pocs.config=={}:
```
Comment (wtgee) [inline bin/pocs_shell]: Same here where you just want to check `if self.pocs.config:`, but I think the point James was making was that if you have not run `setup_pocs` then `self.pocs` will be `None` and the `self.pocs.config` will generate an error. So you first need to check for `pocs` and then check for `config`, i.e. `if self.pocs and self.pocs.config:`

File: bin/pocs_shell [hunk 5]
```diff
@@ -74,6 +74,17 @@ class PocsShell(Cmd):
 
         return self.pocs.is_safe()
 
+    def do_display_config(self, *arg):
+        """ Display the config file for POCS """
+        try:
+            if self.pocs and self.pocs.config:
+                print_warning("No config file for POCS.")
+            else:
```
Comment (jamessynge) [inline bin/pocs_shell]: I think the if and else cases have been swapped.

Comment (jermainegug) [reply bin/pocs_shell]: Seems like I did it for both. Fixing now, thank you.

**Comment by jamessynge** [comment]
Thanks for the feature and fixes.

**Comment by jermainegug** [comment]
I am not familiar with how the `codecov/project` check works, so not sure how to get this check to pass.

**Comment by wtgee** [comment]
The codecov failures don't absolutely prevent us from merging although if they are way off you should pay attention to them. The reports are currently showing some odd things though depending on when it was branched. Don't worry about them for now.

File: bin/peas_shell [hunk 6]
```diff
@@ -8,6 +8,7 @@ import sys
 from astropy.utils import console
 from pprint import pprint
 from threading import Timer
+from pprint import pprint
```
Comment (wtgee) [inline bin/peas_shell]: This still looks like a duplicate of line 9.

**Pull request merged**